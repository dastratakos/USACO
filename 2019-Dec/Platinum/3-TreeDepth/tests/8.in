48 657 654131603
