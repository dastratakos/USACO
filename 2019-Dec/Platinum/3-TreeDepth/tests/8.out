521761918 590764411 362110594 271651449 522825518 614035606 496292487 204525574 30896840 328013007 110745415 37147159 544084974 17707417 615624424 208844937 550741658 496165179 11267815 540257553 416633612 465813784 632831939 318541790 499258264 189205995 383416814 308463326 66063684 284604967 274925790 160196500 100102121 144280784 191800825 158918227 358555026 336949114 400418420 197235857 534065026 382727706 149638897 179398061 5154711 112206928 524726208 537089328
